api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -36,10 +39,23 @@ type IBMVPCMachineSpec struct {
 	// Name of the instance.
 	Name string `json:"name,omitempty"`
 
+	// CatalogOffering is the Catalog Offering OS image which would be installed on the instance.
+	// An OfferingCRN or VersionCRN is required, the PlanCRN is optional.
+	// +optional
+	CatalogOffering *IBMCloudCatalogOffering `json:"catalogOffering,omitempty"`
+
+	// DedicatedHost is the name of the underlying provisioned host in the VPC on which the instance will be created.
+	// +optional
+	DedicatedHost *VPCResource `json:"dedicatedHost,omitempty"`
```

Comment:
Yes, that is the expected field. We will need to keep parity for allowing `DedicateHostIdentities` to be supported for Machines (VSI's).

Reply:
well, this is not limited to Host right, I see following combinations - 
- dedicated host
- dedicated host group
- placement group

So instead of making this field just to host, lets make it more generic so that user can consume for any of the above options..

Correct me if I'm missing something here!

Reply:
I refactored to support the three options for placement target now.